LearningMissionSimulation/PlaygroundEdvard.cs

```diff
@@ -228,12 +258,24 @@ public void CreateClassrooms(int classroomCount)
                     ModuleDictionary.TryGetValue(moduleId, out module);
                     Classroom classroom = ObjectGenerator.GenerateClassroom(module);
                     ClassroomDictionary.Add(classroom.Id, classroom);
+
+                    ReportItem(itemName: classroom.ToString(), actionName: "Generate", itemIndex: ++i);
                 }
             }
+            else
+            {
+                ReportError(missingResource: "Module", failedAction: "Classroom generation");
```

Comment:
You already added the action string, please use it in all report methods that take in actionName.

Reply:
Resolved.